### ISSUE TITLE
Is there any way to exempt the editor's initial state from the undo list? 

### ISSUE THREAD

**Issue by Martin-Alexander** [nhn/tui.editor#3195]
When the editor is loaded with an initial value (for instance, the demo here https://ui.toast.com/tui-editor) issuing an undo command (e.g., with Cmd+Z on macOS) all the content in the editor is deleted.

Most editors and text inputs don't work this way and users seem to find it confusing (and especially frustrating if they're not familiar with the redo shortcut).

Is there a way of preventing this behaviour? Perhaps by clearing the undo history when the editor loads?

**Comment by prostoandrei** [comment]
Yes, it's an API design problem. The only way I got around it was to put onKeyDown processing on the wrapper component.
